createdisk.sh

```diff
@@ -28,9 +28,10 @@ OPENSHIFT_VERSION=$(${JQ} -r .clusterInfo.openshiftVersion $INSTALL_DIR/crc-bund
 # VM_PREFIX: full VM name with the random string generated by openshift-installer
 CRC_VM_NAME=${CRC_VM_NAME:-crc}
 VM_PREFIX=$(get_vm_prefix ${CRC_VM_NAME})
+VM_NAME="${VM_PREFIX}-master-0"
```

Comment:
One of `VM_NAME` or `CRC_VM_NAME` should be renamed, or it's going to be very confusing. `SHORT_VM_NAME` and `FULL_VM_NAME` maybe?
Even `VM_PREFIX=$(get_vm_prefix ${CRC_VM_NAME})` is odd, `VM_PREFIX` is something longer than `CRC_VM_NAME`, I'd expect a prefix of ${CRC_VM_NAME}, ie something shorter.

Reply:
`CRC_VM_NAME=${CRC_VM_NAME:-crc}`  should be `CRC_PRODUCT_NAME=${CRC_PRODUCT_NAME:-crc}` becaue `crc` is just product name and it happened to part of the VM name.

Reply:
Just `PRODUCT_NAME`then if we expect this can be different from `crc`? Or `SNC_PRODUCT_NAME`if you want this namespaced?

Reply:
@cfergeau Added a new commit for this since it happen to be this variable consumed in different helper/script.